### ISSUE TITLE
VoID

### ISSUE THREAD

**Issue by elf-pavlik** [valueflows/valueflows#79]
Vocabulary of Interlinked Datasets
* http://www.w3.org/TR/void/
* http://vocab.deri.ie/void

**Comment by bhaugen** [comment]
I like the name. Void.

**Comment by almereyda** [comment]
Yeah, also had a look into it. Seems a practical starting point for some
cases.

**Comment by pmackay** [comment]
In the interest of consolidating info, would a wiki page listing all "relevant vocab" issues be useful?

**Comment by fosterlynn** [comment]
@pmackay thanks!  But I'm not sure what you mean.  Do you mean issues as in github issues, like a filter on the relevant vocab tag?  Or a list of relevant vocabs themselves?  The latter sounds useful to me!

**Comment by pmackay** [comment]
@fosterlynn I think I meant a wiki page listing the links to the vocabs, which could replace the issues. It could be a reference? Would bring them all together. Also I'm unsure how issues in this repo would get closed (perhaps that relates to #84)?

**Comment by bhaugen** [comment]
Issues in this repo can be closed by any of the valueflows github org [members](https://github.com/orgs/valueflows/people). 

The advantage of the issues are that people can discuss them in place, and add new ones whenever they want, and they can be closed when they are no longer an outstanding issue. And finding all of the issues marked "relevant vocab" is easy, just click on the label on the right panel.

The advantage of a wiki page might be to catalog all of the relevant vocabs on the same page, with some description about how they are relevant, and specify which ones are in use in the valueflows vocab. The vocab could remain in the list after the issue was closed.

**Comment by fosterlynn** [comment]
@pmackay @bhaugen I like the wiki page!  Perhaps they can start as issues if people want to discuss them.  But a reference doc would be great.

**Comment by bhaugen** [comment]
So, @pmackay - were you offering to create such a wiki page? Or just suggesting it as a good idea?

**Comment by pmackay** [comment]
Does https://github.com/valueflows/valueflows/wiki/Useful-vocabularies work as a structure? 

What I'm not sure about is the relevance. Is that intended to be worked through on the issues (in general for the vocabs)?

Where would VoID be used in this case? Are there known datasets that could get published by implementors of VF where it would be relevant?

**Comment by bhaugen** [comment]
@pmackay - I like the structure.  Don't know how we will work between the issues and a wiki table of relevant vocabs.

We have had a lot of discussion about what other vocabs to use, and (I think) have decided we will work through a bunch of examples before making any hard and fast decisions. But I think a table will become useful at some stage: here are the other vocabs we are using, and here's how we are using them.
